client/client.cpp

```diff
@@ -1,23 +1,78 @@
+#include <iostream>
+#include <sodium.h>
+#include <thrift/protocol/TBinaryProtocol.h>
+#include <thrift/transport/TSocket.h>
+#include <thrift/transport/TTransportUtils.h>
+
+#include "../constants/network.h"
 #include "../crypto/encryption_engine.h"
+#include "../gen-cpp/Operation_types.h"
+#include "../gen-cpp/RPC.h"
 #include "../host/redis.h"
-#include <iostream>
-using namespace std;
+
+using namespace apache::thrift;
+using namespace apache::thrift::protocol;
+using namespace apache::thrift::transport;
+
+Operation genRandOperation() {
```

Comment:
(nit) we should use docstrings to document things in the codebase so it's easier to maintain in the future

Reply:
I agree, let's have a discussion on the format for docstrings to keep consistency